proto/v2/markdown.tmpl

```diff
@@ -80,4 +80,4 @@ You can find the source for {{.Name}} [on Github](https://github.com/GoogleConta
 {{end}} <!-- end HasExtensions -->
 
 
-{{end}}
+{{end}}
```

Comment:
nit: is removing the newline required?  Why this change?

Reply:
No newline is removed. Dont know why it captured (removal and addition of newline). Will fix it.